aea/cli/test.py

```diff
@@ -184,6 +179,9 @@ def test_package_by_path(
         "one of either aea_project_path or packages_dir must be specified",
     )
     root_packages = aea_project_path if aea_project_path else packages_dir
+
+    os.environ["PACKAGES_DIR"] = str(root_packages)
```

Comment:
Why?

Reply:
This was a part of an experiment, I'll remove it